### DISCUSSION TITLE
Allow sorting 'dependents' by popularity and other sorting

### DISCUSSION THREAD

**Discussion by jonkoops** [community/community#128415] [npm]
Hi, I maintain several quite popular NPM packages, and one of the things that I often do after a major release is going through packages that depend on my work to see if I can help them upgrade the next version. However the ['dependents' page](https://www.npmjs.com/browse/depended/jwt-decode)  does not offer any way to sort the results. For consistency, it would be really nice to be able to see the same sorting options as a regular search presents:

![image](https://github.com/npm/feedback/assets/695720/b559ee0b-0687-4c5d-9f04-d5991fdc3702)

**Comment by ljharb** [comment]
+1, for this exact same reason.
